File: docflow-backend/src/main/java/com/brutecx/docflow_backend/api/dto/audit/IdentityProjectionAuditCursorPageDTO.java
package com.brutecx.docflow_backend.api.dto.audit;

import java.time.Instant;
import java.util.List;
import java.util.UUID;

public record IdentityProjectionAuditCursorPageDTO(
        List<IdentityProjectionAuditDTO> items,
        boolean hasMore,
        Instant nextCursorTimestamp,
        UUID nextCursorId
) implements BaseAuditCursorPageDTO<IdentityProjectionAuditDTO> {
}

File: docflow-frontend/types/api/IdentityProjectionAuditCursorPageDTO.ts
import { IdentityProjectionAuditRow } from "./IdentityProjectionAuditRow";

export type IdentityProjectionAuditCursorPageDTO = {
  items: IdentityProjectionAuditRow[];
  hasMore: boolean;
  nextCursorTimestamp: string | null;
  nextCursorId: string | null;
};
